rpcs3/Emu/System.cpp

```diff
@@ -408,7 +408,18 @@ void Emulator::Init(bool add_only)
 			}
 		}
 
-		make_path_verbose(dev_hdd0 + "disc/");
+		const std::string games_common_dir = g_cfg_vfs.get(g_cfg_vfs.games_dir, emu_dir);
+
+		if (make_path_verbose(games_common_dir))
+		{
+			fs::write_file(games_common_dir + "/Disc Games Can Be Put Here For Automatic Detection.txt", fs::create + fs::excl + fs::write, ""s);
+
+			if (std::string rpcs3_shortcuts = games_common_dir + "/shortcuts"; make_path_verbose(rpcs3_shortcuts))
+			{
+				fs::write_file(rpcs3_shortcuts + "/Copyable Shortcuts For Installed Games Would Be Added Here.txt", fs::create + fs::excl + fs::write, ""s);
```

Comment:
as far as I can tell you aren't using the vfs in the shortcut creation, so this is a kinda weird file to put there

Reply:
Fixed.